src/main/webapp/app/course/manage/course-update.component.ts

```diff
@@ -598,6 +599,8 @@ export class CourseUpdateComponent implements OnInit {
         this.croppedImage = undefined;
         this.courseForm.controls['courseIcon'].setValue(undefined);
     }
+
+    protected readonly FeatureToggle = FeatureToggle;
```

Comment:
why is this protected?

Reply:
This follows the principle of encapsulation. Since there is no need to access this from outside of the component, it shouldn't  be revealed to the outside.